extensions/blocks/mailchimp/mailchimp.php

```diff
@@ -102,16 +115,47 @@ function jetpack_mailchimp_block_load_assets( $attr ) {
 				<p id="wp-block-jetpack-mailchimp_consent-text" name="wp-block-jetpack-mailchimp_consent-text">
```

Comment:
This seems to be what's triggering the AMP validation error.

Reply:
Should be `class`, no?

Reply:
`name` was probably there to match to the FORM's `aria-describedby` attribute, but the docs say that `id` is sufficient. Removed in https://github.com/Automattic/jetpack/pull/12695/commits/f64a3bfb71e7966a7508a349f6f3bdee541e8834